lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -517,6 +519,18 @@ def verify_paging_flow_rules(self, ip_list):
             )
             assert bool(has_tunnel_action)"""
 
+    def gen_imsi(self, prefix=None):
+        """
+        Generate imsi based on index offset and prefix
+        """
+        idx = str(self._imsi_idx)
+        # Add 0 padding
+        padding = self.IMSI_LEN - len(idx) - len(prefix[4:])
+        imsi = prefix + "0" * padding + idx
+        self._imsi_idx += 1
+        print("Using subscriber IMSI %s" % imsi)
+        return imsi
+
```

Comment:
is this function used anywhere in this PR? might be better to move to the relevant PR.